app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,40 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @foods = current_user.foods.order(created_at: :desc)
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = current_user.foods.build(food_params)
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to foods_url, notice: 'Food was successfully created.' }
+      else
+        format.html { render :new, alert: 'Food could not be created' }
+      end
+    end
+  end
+
+  def destroy
+    @food.destroy
+    respond_to do |format|
+      format.html { redirect_to foods_url, notice: 'Food was successfully destroyed.' }
+    end
+  end
+
+  private
+
+  def set_food
+    @food = Food.find(params[:id])
+  end
+
+  def food_params
+    params.require(:food).permit(:name, :measurement_unit, :quantity, :price)
+  end
+end
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you did, but I couldn't use the `Add Ingredient` feature because I needed to create a `Food` to use it, but I couldn't create it because there was an `error` in the `Add Food` section. Please fix it 🙏🏼 

##
![image](https://user-images.githubusercontent.com/63915024/226019029-bc0a262e-fe99-42a5-898d-f5758a6e9ba9.png)
##
![image](https://user-images.githubusercontent.com/63915024/226019159-d2da6bef-bbdd-4a30-ad2e-8de3359ffe9d.png)
##

Reply:
Thank you @mukhammadrizooff , we will fix the issue!